lib/rubysl/rubysl-strscan/lib/rubysl/strscan/strscan.rb

```diff
@@ -177,7 +179,7 @@ def matched?
   end
 
   def matched_size
-    @match.to_s.bytesize if @match
+    @width if @match
```

Comment:
I think it would be simpler to just have a primitive on match data like `:matchdata_match_bytesize`, than adding extra state around the class.

Reply:
Actually it could be just `@match.byte_end(0) - @match.byte_begin(0) if @match` since we have those methods on MatchData.